synapse/models/transport.py

```diff
@@ -52,11 +106,169 @@ def getModelDefs(self):
+                    'interfaces': ('transport:vehicle',),
+                    'template': {'phys:object': 'train'},
+                    'doc': 'A group of rail cars and locomotives connected together.'}),
+
+                ('transport:shipping:container', ('guid', {}), {
+                    'interfaces': ('transport:container',),
+                    'template': {'phys:object': 'shipping container'},
+                    'doc': 'An individual shipping container.'}),
+
+            ),
+            'interfaces': (
+
+                ('transport:container', {
+                    'interfaces': ('phys:object',),
+                    'doc': 'Properties common to a container used to transport cargo or people.',
+                    'props': (
+
+                        ('built', ('time', {}), {
+                            'doc': 'The date when the {phys:object} was built.'}),
+
+                        ('manufacturer', ('ou:org', {}), {
+                            'doc': 'The organization which manufactured the {phys:object}.'}),
+
+                        ('manufacturer:name', ('ou:name', {}), {
+                            'doc': 'The name of the organization which manufactured the {phys:object}.'}),
+
+                        ('model', ('str', {'lower': True, 'strip': True}), {
+                            'doc': 'The model of the {phys:object}.'}),
+
+                        ('serial', ('str', {'strip': True}), {
+                            'doc': 'The manufacturer assigned serial number of the {phys:object}.'}),
+
+                        ('max:occupants', ('int', {'min': 0}), {
+                            'doc': 'The maximum number of occupants the {phys:object} can hold.'}),
+
+                        ('max:cargo:mass', ('mass', {}), {
+                            'doc': 'The maximum mass the {phys:object} can carry as cargo.'}),
+
+                        ('max:cargo:volume', ('geo:dist', {}), {
+                            'doc': 'The maxiumum volume the {phys:object} can carry as cargo.'}),
+
+                        # TODO deprecate for entity:ownership?
+                        ('owner', ('ps:contact', {}), {
+                            'doc': 'The contact information of the owner of the {phys:object}.'}),
+                    ),
+                }),
+                # most containers are vehicles, but some are not...
+                ('transport:vehicle', {
+                    'interfaces': ('transport:container',),
+                    'templates': {'phys:object': 'vehicle'},
+                    'doc': 'Properties common to a vehicle.',
+                    'props': (
+                        ('operator', ('ps:contact', {}), {
+                            'doc': 'The contact information of operator of the {phys:object}.'}),
+                    ),
+                }),
+
+                ('transport:schedule', {
+                    'doc': 'Properties common to travel schedules.',
```

Comment:
Would we still use transport:schedule in the event that we are modeling a flight (for example), that was supposed to take off at 8am and last 3 hours and arrive at 12pm local time, but was ultimately cancelled? In that case, would we just capture the :scheduled:* prop values and leave :departed:* and :arrived:* empty? Or would we want something like a status property to note that this was scheduled but cancelled?

Actually based on the below it looks like we'd model the transport:schedule but then use transport:segment:status to note that it was cancelled?